pkgs/development/haskell-modules/configuration-ghc-9.2.x.nix

```diff
@@ -63,6 +63,7 @@ self: super: {
   HTTP = overrideCabal (doJailbreak super.HTTP) (drv: { postPatch = "sed -i -e 's,! Socket,!Socket,' Network/TCP.hs"; });
   integer-logarithms = overrideCabal (doJailbreak super.integer-logarithms) (drv: { postPatch = "sed -i -e 's,integer-gmp <1.1,integer-gmp < 2,' integer-logarithms.cabal"; });
   lukko = doJailbreak super.lukko;
+  network = doJailbreak super.network;
```

Comment:
Wouldn't using `network_3_1_2_2` work as well?

Reply:
I didn't check since the jailbreak worked immediately.

I just built `network_3_1_2_2`, that works as well (it's a version bump release ^^) - I'll swap it.